packages/openassistants/openassistants/llm_function_calling/utils.py

```diff
@@ -1,23 +1,20 @@
 import json
+from typing import Optional
 
 import numpy as np
 from langchain.chat_models.base import BaseChatModel
 from langchain.chat_models.openai import ChatOpenAI
 from langchain.schema.messages import SystemMessage
 
 from openassistants.data_models.chat_messages import ensure_alternating
-from openassistants.utils.langchain import is_openai
+from openassistants.utils.langchain_util import is_openai
 
 OUTPUT_FORMAT_INSTRUCTION = """\
-The output should be formatted as a JSON instance that conforms to the JSON schema below.
-
-As an example, for the schema {{"properties": {{"foo": {{"title": "Foo", "description": "a list of strings", "type": "array", "items": {{"type": "string"}}}}}}, "required": ["foo"]}}
-
-the object {{"foo": ["bar", "baz"]}} is a well-formatted instance of the schema. The object {{"properties": {{"foo": ["bar", "baz"]}}}} is not well-formatted.
+The output should fit the JSON Schema:
```

Comment:
I think I want to stick to the prompt we previously used as it's based on LangChain's generic Pydantic parser which I think has gotten more testing than ours (I don't think we tested quantitatively what worked better, right?). Note this is not used when OpenAI Functions are available.